src/common/containers/landing.js

```diff
@@ -125,7 +125,7 @@ class Landing extends Component {
               </div>
             </div>
             <div className={styles.oneThird}>
-              <img src='https://assets.ubuntu.com/v1/2c5e93c5-fast-easy-safe-illustration.svg'/>
+              <img alt="" src='https://assets.ubuntu.com/v1/2c5e93c5-fast-easy-safe-illustration.svg'/>
```

Comment:
@richmccartney This is a different kind of image from the previous ones. [To quote the HTML spec once more](https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn%27t-add-any-information), it is “a decorative image that isn't discussed by the surrounding text but still has some relevance … In these cases, the `alt` attribute must be present but its value must be the empty string.”